updates/update-1.1.0.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-}
+use SwedbankPay\Payments\WooCommerce\WC_Swedbank_Pay_Transactions;
+
+defined( 'ABSPATH' ) || exit;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of exit language construct is discouraged.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)